src/structures/Inode.cpp

```diff
@@ -0,0 +1,83 @@
+#include <iostream>
+#include <vector>
+#include "../includes/Inode.hpp"
+
+Inode::Inode(bool src_type):
+            src_type(src_type),
+            number_references(0),
+            sizeof_file(0),
+            last_access_time(time(nullptr)),
+            last_file_modif_time(time(nullptr)),
+            last_inode_modif_time(time(nullptr)),
+            blocks_amount(1) { }
+
+
+Inode::Inode(bool src_type, size_t address_block):
+            src_type(src_type),
+            number_references(0),
+            sizeof_file(0),
+            last_access_time(time(nullptr)),
+            last_file_modif_time(time(nullptr)),
+            last_inode_modif_time(time(nullptr)),
+            blocks_amount(1)
+{
+    blocks_storage.push_back(address_block);
+}
+
+// Maybe another type
+const std::vector<size_t>& Inode::get_blocks_storage() {  
+    return blocks_storage;
+}
+
+size_t Inode::get_last_block(){
+    return blocks_storage.back();
+}
+
+// Print all fields of inode 
+void Inode::print_fields() { 
+    std::cout << "Magic number of inode = " << magic_number << std::endl;
+    std::cout << "optional_bit1 = " << optional_bit1 << std::endl;
+    std::cout << "optional_bit2 = " << optional_bit2 << std::endl;
+    std::cout << "Number of references to file = " << number_references << std::endl;
+    std::cout << "Identificator of owner/user and group-owner = " << identifier << std::endl;
+    std::cout << "Current sizeof file = " << sizeof_file << std::endl;
+    std::cout << "Time of the last access to the file = " << last_access_time << std::endl;
+    std::cout << "Time of the last file modification = " << last_file_modif_time << std::endl;
+    std::cout << "Time of the last inode modification = " << last_inode_modif_time << std::endl;
+    std::cout << "Current sizeof array with storage block addresses = " << blocks_amount << std::endl;
+    std::cout << "Array of storage block addresses: ";
+    for(int i = 0; i < blocks_amount; i++) std::cout << blocks_storage[i] << " ";
+}
```

Comment:
it's cosmetic comment, but anyway. You don't need to close stream for every line. It's enough to do this only for last line. And yeah, you didn't closed it.